src/renderer/utils.js

```diff
@@ -35,30 +33,30 @@ function hex2rgba(hex) {
 }
 
 function getOptions(options) {
-  if (!options) options = {};
-  if (!options.color) options.color = {};
+  const resolvedOptions = options || {};
+  if (!resolvedOptions.color) resolvedOptions.color = {};
 
   const margin =
-    typeof options.margin === "undefined" ||
-    options.margin === null ||
-    options.margin < 0
+    typeof resolvedOptions.margin === "undefined" ||
+    resolvedOptions.margin === null ||
+    resolvedOptions.margin < 0
       ? 4
-      : options.margin;
+      : resolvedOptions.margin;
```

Comment:
Instead of use `resolvedOptions` var you could rename the param name to `opts` so the first line could be:

```js
const options = opts || {}; 
```